pages/index.tsx

```diff
@@ -1,66 +1,17 @@
+import Layout from 'components/Layout';
 import Head from 'next/head';
 import styles from '../styles/Home.module.css';
 
 const Home: React.FC = () => {
   return (
-    <div className={styles.container}>
+    <Layout title="Home Page | Next.js + TypeScript Example">
       <Head>
-        <title>Create Next App</title>
+        <title>NBJC Home</title>
```

Comment:
sorry, you don't need the `<Head>` tag because the `<Layout>` component has in inside